lib/generators/manageiq/plugin/templates/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "<%= plugin_name %>",
+  "version": "1.0.0",
+  "description": "ManageIQ Cloud Management Platform",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ManageIQ/manageiq.git"
+  },
+  "author": "ManageIQ",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/ManageIQ/manageiq/issues"
+  },
+  "homepage": "https://github.com/ManageIQ/manageiq#readme",
+  "engines": {
+    "node": "<%= node_version %>",
+    "npm": "<%= npm_version %>"
+  },
```

Comment:
NOTE I removed yarn from here because we want to use the version number elsewhere and the `>=` was causing issues there.  Alternatively I could pull the entire `"engines"` section from ui-classic and just use that as is

Reply:
This is fine.